.github/workflows/CI-build.yml

```diff
@@ -0,0 +1,35 @@
+name: CI build
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    
+      - name: checkout dbeaver project
+        uses: actions/checkout@v2
+        with:
+          repository: dbeaver/dbeaver
+          path: dbeaver        
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build dbeaver with maven
+        working-directory: dbeaver/
+        run: mvn clean install
+    
+      - name: checkout orbisrcp project
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build orbisrcp with Maven
+        run: mvn clean install
+        
+      - name: Unzip p2-1.0.0-SNAPSHOT.zip
+        uses: TonyBogdanov/zip@1.0
```

Comment:
what ?

Reply:
This is just to check the "mvn clean install" generate this zip file we use in DBeaver.